cinema/tests/test_movie_api.py

```diff
@@ -43,9 +42,7 @@ def sample_actor(**params):
 
 
 def sample_movie_session(**params):
-    cinema_hall = CinemaHall.objects.create(
-        name="Blue", rows=20, seats_in_row=20
-    )
+    cinema_hall = CinemaHall.objects.create(name="Blue", rows=20, seats_in_row=20)
```

Comment:
It's a good practice to separate test data creation into a separate method or use a factory library like Factory Boy. This way, you can avoid code duplication and make your tests cleaner and easier to maintain.